admin/partials/settings/compressor.php

```diff
@@ -53,6 +53,29 @@
 }
 $select_compressor .= '</select>';
 
+// Create the Compression level options.
+$default_compression_level  = 6;
```

Comment:
This code is duplicated elsewhere. Just call that other code to get the compression level.